src/schemas/index.ts

```diff
@@ -0,0 +1,3 @@
+import SchemaForm from '@/schemas/components/SchemaForm.vue'
+
+export { SchemaForm as SchemaFormV2 }
```

Comment:
Exporting as SchemaFormV2 to not conflict with the existing SchemaForm component